release.json

```diff
@@ -7,7 +7,7 @@
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
         "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "OMNIBUS_RUBY_VERSION": "chouquette/fix_deb_compression_level",
```

Comment:
Just double checking - is this the intended `OMNIBUS_RUBY_VERSION`? Asking because it changed from `datadog` to what seems to be a personal version / level.

Reply:
I think it indicates a branch for testing, but it will be reverted before merging
The `gitlab/do-not-merge` [job](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/404117458#L86) prevents merging with a non standard value by the way

Reply:
Pierre's correct, this will also require a change in omnibus-ruby to be fully fixed. This PR will need to be updated once that other PR has been merged and we can revert back to the regular branch

Thanks for double checking!